xcm/xcm-builder/src/lib.rs

```diff
@@ -97,5 +97,8 @@ pub use universal_exports::{
 	SovereignPaidRemoteExporter, UnpaidLocalExporter, UnpaidRemoteExporter,
 };
 
+mod origin_aliases;
+pub use origin_aliases::AliasForeignAccountId32;
+
 mod pay;
-pub use pay::{FixedLocation, LocatableAssetId, PayAccountId32OnChainOverXcm, PayOverXcm};
+pub use pay::{FixedLocation, LocatableAssetId, PayAccountId32OnChainOverXcm, PayOverXcm};
```

Comment:
Needs a newline here:
```suggestion
pub use pay::{FixedLocation, LocatableAssetId, PayAccountId32OnChainOverXcm, PayOverXcm};

```